assemblies/common-resources/src/main/resources-filtered/etc/custom.properties

```diff
@@ -184,7 +184,8 @@ org.osgi.framework.system.packages.extra= \
  org.h2.*, \
  com.cloudera.impala.jdbc41, \
  org.postgresql.*, \
- org.pentaho.di.core.namedcluster.*;version="${big-data-plugin.version}"
+ org.pentaho.di.core.namedcluster.*;version="${big-data-plugin.version}", \
+ com.hitachivantara.security.web.*;version="${hv-security-web.version}"
```

Comment:
Changes are only for pentaho server, remove the package export from here as this affects other assemblies such as PDI.